mypy/build.py

```diff
@@ -876,6 +876,20 @@ def write_cache(id: str, path: str, tree: MypyFile,
         data_str = json.dumps(data, sort_keys=True)
     interface_hash = compute_hash(data_str)
 
+    # Obtain and set up metadata
+    try:
+        st = manager.get_stat(path)
+    except OSError as err:
+        manager.log("Cannot get stat for {}: %s".format(path, err))
```

Comment:
Something is wrong with formatting here, this does not fail, but ``%s`` is never replaced (at least in Python 3.4 I just tried).